tests/unit/bokeh/embed/test_bundle.py

```diff
@@ -133,6 +133,22 @@ def test_with_Server_resources(self) -> None:
         plot = Plot()
         plot.add_layout(LatexLabel())
         bundle = beb.bundle_for_objs_and_resources([plot], "server")
+        version_hash = "6b13789e43e5485634533de16a65d8ba9d34c4c9758588b665805435f80eb115"
+        assert len(bundle.js_files) == 2
+        assert (bundle.js_files[1] ==
+                f"http://localhost:5006/static/extensions/latex_label/latex_label.js?v={version_hash}")
+
+    def test_with_Server_resources_dev(self) -> None:
+        from latex_label import LatexLabel
+        plot = Plot()
+        plot.add_layout(LatexLabel())
+        try:
+            os.environ['BOKEH_RESOURCES'] = 'server'
+            os.environ['BOKEH_DEV'] = 'True'
+            bundle = beb.bundle_for_objs_and_resources([plot], "server")
+        finally:
+            del os.environ['BOKEH_DEV']
+            del os.environ['BOKEH_RESOURCES']
```

Comment:
I would be better to preserve the original.

Reply:
Originally did that but noticed that a bunch of other tests didn't do that. Would also be surprised if running tests with these environment variables set is a good idea. Happy to make the change anyway.

Reply:
It's not a good idea and tests will fail in all possible ways, but at least they will fail consistently, independently of what subset of tests is run. However, seeing that this is a common pattern already, let's ignore this for now and fix all occurrences in another PR.

Reply:
Should probably ship a context manager to safely set and unset the environment variables somewhere.

Reply:
I started issue #11574 for this.